src/components/Apps/AccountListTable/const.js

```diff
@@ -234,7 +234,7 @@ export const accountQuickFilters = (vm) => [
         }
       },
       {
-        label: vm.$t('LongTimeNoChangeSecret'),
+        label: vm.$t('LongTimePassword'),
         filter: {
           long_time_no_change_secret: 'true'
         }
```

Comment:
I cannot determine the differences in coding without seeing the code itself. Please attach or describe the files you would like to compare here so I can assist you with your concerns.